.changeset/nice-peas-marry.md

```diff
@@ -0,0 +1,10 @@
+---
+"@hopper-ui/svg-icons": patch
```

Comment:
Have you been running `generate-icons` from the root or from the project? 

When  i go in your branch, if i generate-icons from the root, which runs generate-icons on svg-icons, then generate-icons in icons, i get 33 new changes.

Reply:
They weren't generated at the root of the project, so I'm going to validate the changes and adjust the taffy as needed.